Change a lot of code for new functions of ES6

jsanchez91 opened the pull request:
The button put after the input:password

I hope you like it

jesuslg123 commented on .vscode/launch.json:
```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense para saber los atributos posibles.
```
Please remove comments :)

jesuslg123 commented on backup.txt:
```diff
@@ -0,0 +1 @@
+eyIxIjp7IklEIjoxLCJuYW1lIjoiTXkgaWNvbnMgY29sbGVjdGlvbiIsImJvb2ttYXJrX2lkIjoiajV0bTlvcjNuaDAwMDAwMCIsImNyZWF0ZWQiOm51bGwsInVwZGF0ZWQiOjE1MzkyNTc4NzksImFjdGl2ZSI6MSwic291cmNlIjoibG9jYWwiLCJvcmRlciI6MCwiY29sb3IiOiIwMDAwMDAiLCJzdGF0dXMiOjF9LCJqNXRtOW9yM25oMDAwMDAwIjpbeyJpZCI6MTgxNTMzLCJ0ZWFtIjowLCJuYW1lIjoibG9jayIsImNvbG9yIjoiIzAwMDAwMCIsInByZW1pdW0iOjAsInNvcnQiOjF9LHsiaWQiOjE4MTUzNCwidGVhbSI6MCwibmF ...
```
What its this file for? Need to be committed?

jesuslg123 commented on script.js:
```diff
@@ -1,7 +1,6 @@
-console.log("hi!")
```
Thx :)

jesuslg123 reviewed (changes requested):
Nice job! Please just check the comments before merge.

jsanchez91 commented:
It's time to accept my changes :wink:

jesuslg123 commented on script.js:
```diff
@@ -1,45 +1,75 @@
-	button.onclick = function(e) {
+	var element_style = window.getComputedStyle(element)
+	IMG_LOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/lock.png')+'">'
+    IMG_UNLOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/unlock.png')+'">'
```
Indentation went crazy here xD

jsanchez91 replied:
Solved

jsanchez91 committed:
Code improvement

Change all the functions by the new functions of ES6 Javascript.
Added two images for the buton

jsanchez91 committed:
Solved identation problem

jsanchez91 commented:
Solved the problem

jesuslg123 commented on script.js:
```diff
@@ -1,45 +1,75 @@
-		if (type == 'password') {
-			processInput(input)
-		}
+	var inputs = document.querySelectorAll('input[type="password"]')
```
Nice change 💯

jesuslg123 commented on script.js:
```diff
@@ -1,45 +1,75 @@
-	button.onclick = function(e) {
+	var element_style = window.getComputedStyle(element)
+	IMG_LOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/lock.png')+'">'
+        IMG_UNLOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/unlock.png')+'">'
```
Now looks deeper than necesary xD IDE is going crazy? Don't worry :P

jesuslg123 commented on script.js:
```diff
@@ -1,45 +1,75 @@
+	var element_style = window.getComputedStyle(element)
+	IMG_LOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/lock.png')+'">'
+        IMG_UNLOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/unlock.png')+'">'
+	console.log(element_style.paddingTop)
```
Do we need to log the padding?

jesuslg123 reviewed (approved):
Awesome work :)
